shadows/framework/src/main/java/org/robolectric/shadows/ShadowLegacyTypeface.java

```diff
@@ -161,28 +162,25 @@ public int hashCode() {
 
   @HiddenApi
   @Implementation
-  protected static Typeface createFromFamilies(Object /*FontFamily[]*/ families) {
+  protected static Typeface createFromFamilies(FontFamily[] families) {
```

Comment:
emm, so I should still use ClassName annotation with Object type instead of using FontFamily type directly such as `protected static Typeface createFromFamilies(FontFamily[] families)`?

Reply:
Hi @utzcoz,
still confused why we need ClassName annotation here, since `protected static Typeface createFromFamilies(FontFamily[] families)` in this shadow is marked HiddenApi and protected, looks like other developer will not be able to access this function directly? thus no way to access FontFamily class?

Reply:
Hi @utzcoz ,
thanks for the detailed explanation, very informative.
I am still confused why not use the type explicitly. so
1) `FontFamily was added in AOSP when they develop Android SDK 21` 
2)  `protected` modifier
These two will guarantee 
1) Class resolution will be good since the class is indeed in the sdk
2) Normal developers will not be able to call `ShadowLegacyType#createFromFamilies` directly since it is protected?